### PR TITLE
Fixed link to the https://github.com/google/re2/wiki/Syntax

### PR DESCRIPTION
**Pull request by eabykov**
Removed `)` in the end of the link https://github.com/google/re2/wiki/Syntax

Fixes: https://github.com/istio/istio.io/pull/15231

Related to: https://github.com/istio/api/pull/3227

### CODE REVIEW

**Comment by eabykov** [comment]
@kfaseela finally: done

**Comment by kfaseela** [comment]
i think there will be some more files. but anyways

File: networking/v1alpha3/virtual_service.proto [hunk 1]
```diff
@@ -738,7 +738,7 @@ message HTTPMatchRequest {
   //
   // - `prefix: "value"` for prefix-based match
   //
-  // - `regex: "value"` for RE2 style regex-based match (https://github.com/google/re2/wiki/Syntax).
+  // - `regex: "value"` for RE2 style regex-based match https://github.com/google/re2/wiki/Syntax
```
Comment (kfaseela) [inline networking/v1alpha3/virtual_service.proto:741]: we should retain the brackets by escaping the character I think

Comment (eabykov) [reply networking/v1alpha3/virtual_service.proto:741]: If you believe that this is the best course of action, then please proceed. I am tired of attempting to assist you with this broken link (and I now understand why no one has attempted this before me).

Comment (kfaseela) [inline networking/v1alpha3/virtual_service.proto:741]:
```suggestion
  // - `regex: "value"` for [RE2 style regex-based match](https://github.com/google/re2/wiki/Syntax)
```

Comment (kfaseela) [reply networking/v1alpha3/virtual_service.proto:741]: If you would like to proceed with the PR, you can do like above at all occurrences of this link, and then do a `make gen` to generate all the required files including the html. Then only the gen check will pass on the PR. But if you do not want to do it, let me know, I can push a separate PR

**Comment by eabykov** [comment]
thx

**Pull request closed without merging**